config/default.php

```diff
@@ -205,6 +205,13 @@
 $raidApiKey = '';                                                   // Raid API Key, '' to deny access
 $sendRaidData = false;                                              // Send Raid data, false to only send gym data
 
+//-----------------------------------------------------
+// Areas
+//-----------------------------------------------------
+
+$noAreas = false;
+$areas = [];                                                   // [[latitude,longitude,zoom,name],[latitude,longitude,zoom,name]]
```

Comment:
In the comment add quotes around `name` to make sure it is added correctly.

Reply:
fixed